GHzDACs/Cleanup/dac.py

```diff
@@ -677,8 +677,25 @@ def _setPolarity(self, chan, invert):
 
     @inlineCallbacks
     def _checkPHOF(self, op, fifoReadings, counterValue):
-        # Determine for which PHOFs (FIFO offsets) the FIFO counter equals
-        # counterValue.
+        """ Determine correct PHOF for FIFO counter of counterValue
+
+        If finds such a PHOF, sets PHOF and reads back FIFO counter to
+        verify that FIFO counter equals counterValue.
+
+        If no PHOF can be found to get an acceptable FIFO counter, PHOF will
+        be returned as -1, and success will be False.
+
+        :param int op: 2 for DAC A, 3 for DAC B
+        :param list[int] fifoReadings: FIFO counters for each PHOF
+                                       (FIFO phase offset)
+        :param int counterValue: Desired FIFO counter.
+        :return: PHOF - PHOF (phase offset) required for targetFifo.
+                        If not successful, -1.
+                 found - If procedure successful. Requires PHOF exists
+                         for FIFO counter to be counterValue and, after
+                         setting PHOF, that FIFO counter is counterValue
+        :rtype: (int, bool)
+        """
```

Comment:
I think it would be better to define PHOF (as phase offset? FIFO phase offset?) when it's first introduced, after the first line. Since we already know that the goal is to find the valid PHOF, I would condense the :return: part to:
```
:return: (PHOF for given counterValue, success)
```